packages/core-utils/src/itinerary.ts

```diff
@@ -410,10 +410,10 @@ export function getCompanyForNetwork(
  * @return {string}  A label for use in presentation on a website.
  */
 export function getCompaniesLabelFromNetworks(
-  networks: string[],
+  networks: string[] | string,
```

Comment:
Is there a reason we can't make a single item array where this method is called rather than polluting the argument types?

Reply:
Yes it's because I don't understand why or when we get a string instead of an array